COLLABORATOR_GUIDE.md

```diff
@@ -444,6 +444,10 @@ commit logs, ensure that they are properly formatted, and add
 * The commit message text must conform to the
 [commit message guidelines](./CONTRIBUTING.md#step-3-commit).
 
+Run tests (`make test` or `vcbuild test`). Even though there was a successful
```

Comment:
Thanks for the Windows inclusion. We are people too 😉 (maybe foolish but people).
Also #12281 `vcbuild test` can easily take 30 minutes after even a minor change.